### PR TITLE
Fix Configuration regarding new changes

### PR DESCRIPTION
**Pull request by DavDo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-1684

**Additional info:**
Changes based on - https://github.com/salemove/mobile-sdk-unified/pull/19

MOB 1684

### CODE REVIEW

**Comment by DavDo** [comment]
!squash

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/base/BadgeTheme.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.glia.widgets.view.unifiedui.theme.base
+
+internal data class BadgeTheme(
+    val textColor: ColorTheme?,
+    val background: LayerTheme?,
+    val textSize: Float?, // Size in SP
+    val textStyle: Int?, //Typeface.NORMAL
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/base/BadgeTheme.kt:7]: Minor comment. To keep it perfectly nice, let's add a space after `//` as you did in the line above.

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/ViewExstensions.kt [hunk 2]
```diff
@@ -130,6 +130,7 @@ internal fun View.applyLayerTheme(layer: LayerTheme?) {
 }
 
 internal fun MaterialCardView.applyCardLayerTheme(layer: LayerTheme?) {
+    layer?.fill?.primaryColor?.also { if (Color.alpha(it) > 0) cardElevation = 0f }
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/ViewExstensions.kt:133]: Could you please explain the logic here? If the color has opacity then we remove card elevation? What's the purpose of this?

**Review by andrews-moc — APPROVED** [review]

**Review by AndriiHorishniiMOC — APPROVED** [review]

**Comment by DavDo** [comment]
!squash

**Comment by DavDo** [comment]
!squash

**Pull request merged**